keyboards/3.0/pcm.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE keyboard SYSTEM '../dtd/ldmlKeyboard.dtd'>
+<keyboard locale="pcm" conformsTo="techpreview">
+	<version number="1.0.0" />
+	<names>
+		<name value="Nigerian Pidgin" />
+		<name value="Naijíriá Píjin" />
```

Comment:
Yeah even though we said the implementation can pick a name it wants (or the first one), I feel like it would be useful to somehow note down what these values are, or how they differ, similar to `rel` attribute in `link`

Reply:
:+1: `rel` attribute would be very helpful.

Reply:
great idea, file a ticket ? :)

Reply:
@miloush do you want to file a ticket on this one?